src/katello/deb_tracer.py

```diff
@@ -27,45 +27,42 @@ def use_flag():
 
 
 def needrestart():
-    process = subprocess.Popen(["needrestart", "-b"], stdout=subprocess.PIPE)
-    output = process.communicate()
-    return output[0].decode().split("\n")
+    output = subprocess.check_output(["needrestart", "-b"])
+    return output.decode().split("\n")
 
 
 def use_needrestart():
     apps = []
     services = []
-    kasta = None
```

Comment:
it's probably still good to define it. If `output` is empty or if `NEEDRESTART-KSTA` is not in the output, line 47 will fail on an undefined variable